docs/guides/installation.rst

```diff
@@ -1,36 +1,56 @@
 Installation
 ============
 
-This section describes how to install `HTSinfer` using Conda.
+This section describes how to install `HTSinfer`.
 
-Clone the Repository and Install Dependencies
----------------------------------------------
+Install using Conda
+-------------------
 
-To install `HTSinfer`, first clone the repository and install the dependencies via `Conda <https://docs.conda.io/en/latest/miniconda.html>`_:
+The easiest and quickest installation method is via `Mamba <https://mamba.readthedocs.io/en/latest/installation.html>`_ or `Conda <https://docs.conda.io/en/latest/miniconda.html>`_.
+`HTSinfer` is available as part of the `Bioconda <https://anaconda.org/bioconda/htsinfer>`_ channel.
+
+To create a new Conda environment with `HTSinfer` and its dependencies installed, run:
+
+.. code-block:: bash
+
+   mamba create --name htsinfer bioconda::htsinfer
+
+
+Then, activate the `htsinfer` Conda environment with:
+
+.. code-block:: bash
+
+   conda activate htsinfer
+
+
+To install `HTSinfer` in your current environment, run:
+
+.. code-block:: bash
+
+   mamba install bioconda::htsinfer
+
+Install from GitHub
+-------------------
+
+If you would like to contribute to the development of `HTSinfer`, or wishing to use unreleased versions, you can install `HTSinfer` from the `GitHub <https://github.com/zavolanlab/htsinfer>`_.
```

Comment:
Maybe to put the more common use case first. Also maybe better to split up cloning and installing, and put two separate code blocks for the alternatives (with or without dev deps):

---

If you wish to use the latest unreleased version of the tool or if you would like to contribute to its further development, please install _HTSinfer_ from [GitHub](https://github.com/zavolanlab/htsinfer).

First clone the repository:

```sh
git clone https://github.com/zavolanlab/htsinfer.git
cd htsinfer
```

Then, install _HTSinfer_ and its dependencies via [Conda](https://docs.conda.io/en/latest/miniconda.html):

```sh
conda env create --file environment.yml
```

Alternatively, to include _HTSinfer_'s optional development dependencies, run the following instead:

```sh
conda env create --file environment-dev.yml
```

After completing the installation, activate the resulting `htsinfer` Conda environment with:

```
conda activate htsinfer
```